test/rebar3_hex_integration_SUITE.erl

```diff
@@ -119,22 +123,59 @@ docs_test(Config) ->
     P = #{app => "valid", mocks => [docs]},
     {ok, #{rebar_state := State, repo := Repo}} = setup_state(P, Config),
     Command = ["docs"],
-    {ok, DocState} = test_utils:mock_command(rebar3_hex_docs, Command, Repo, State),
+    RepoConfig =  [{repos,[Repo]}],
+    {ok, DocState} = test_utils:mock_command(rebar3_hex_docs, Command, RepoConfig, State),
     {ok, NewState} = rebar_prv_edoc:do(DocState),
     ?assertMatch({ok, NewState}, rebar3_hex_docs:do(NewState)).
 
+docs_post_process_test(Config) ->
+    P = #{app => "valid", mocks => [docs]},
```

Comment:
🤔 This always fail with a rebar_abort exception, tested outside of test suite, works fine. Maybe `rebar_utils:sh/2` can't reliably be used in common tests?